apps/charterafrica/scripts/ecosystem.js

```diff
@@ -0,0 +1,18 @@
+const { exec } = require("child_process");
+
+require("dotenv").config({ path: "./.env" });
+require("dotenv").config({ path: "./.env.local" });
+
+(async () => {
+  const { RESOURCES_SECRET_TOKEN } = process.env;
+  const { NEXT_PUBLIC_APP_URL } = process.env;
+  const METHOD = process.argv[2];
+  const command = `curl -X ${METHOD} -H "x-api-key: ${RESOURCES_SECRET_TOKEN}" ${NEXT_PUBLIC_APP_URL}/api/v1/resources/ecosystem/entities`;
+  exec(command, (err, stdout) => {
+    if (err) {
+      console.error(err);
+      return;
+    }
+    console.log(stdout);
+  });
+})();
```

Comment:
Why do we need IIFE here?

Reply:
@kilemensi this file is going to be executed via package json script so I think having it as an invoked function makes sense

Reply:
Could be but I'm looking at a bunch of scripts in say MUI [repo](https://github.com/mui/material-ui/blob/master/scripts/) & they don't use IIFE.